akka-docs/src/main/paradox/cluster-sharding.md

```diff
@@ -274,16 +276,19 @@ shard resolution, e.g. to avoid too fine grained shards. Once a shard's location
 the only overhead is sending a message via the `ShardRegion` rather than directly.
 
 <a id="cluster-sharding-mode"></a>
-## Distributed Data vs. Persistence Mode
+## Sharding State Store Modes
```

Comment:
I'm always somewhat on the fence if we should include documentation for deprecated features at all. We don't want new users to use it anyway, and existing users probably already know how to use it (or can look at old docs).

If it's a big feature with potentially many users I guess it makes sense to still include deprecated features in docs with clear warnings. Maybe this belongs into that category. Let's keep it in docs.

Reply:
My thoughts, which cycled on this many, many times as I re-read the relevant docs many times, are the same. I'm only "on the fence" about it because there's this mini @raboof on my shoulder warning that until fully replaced, existing users may need it.

My gut always says we should not include docs for deprecated features. Not including them at this point helps make it clear to not use or learn to use it. So let's remove.

Reply:
Particularly since older docs are available and will have it.